packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -41,7 +39,8 @@ export function augmentForDefaultView(indexFields)
   return combineIndexWithDefaultViewIndex({
     viewFields: indexFields,
     prefix: {status:1},
-    suffix: { _id:1, isFuture:1, draft:1, meta:1, groupId:1, af:1, isEvent:1, unlisted:1, postedAt:1, baseScore:1 },
+    authorIsUnreviewed: {$in: [false,null]},
```

Comment:
Not sure what this line is doing here, but it shouldn't be

Reply:
Ah, I couldn't actually remember why I did that and whether it was a mistake but figured you'd mention it if it were. :P